src/docs/product/performance/metrics.mdx

```diff
@@ -95,4 +95,31 @@ The calculation method determines if duration is defined as the entire length of
 
 ## Custom Performance Metrics
 
-Custom performance metrics on transactions allow you to define metrics (beyond the ones mentioned above) that are important to your application and send them to Sentry. Currently, you can set these in the [JavaScript (version `7.0.0` or higher)](/platforms/javascript/performance/instrumentation/performance-metrics/) and [Python (version `1.5.12` or higher)](/platforms/python/performance/instrumentation/performance-metrics/) SDKs.
+<Note>
+
+Although you can set custom metrics in the SDK, the ability to see the data generated by these metrics is is available only if you're in the Early Adopter program and on a plan that includes Performance Monitoring. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony. If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
+
+</Note>
+
+In addition to the automatic performance metrics described above, Sentry supports setting custom performance metrics on transactions. Custom performance metrics allow you to define metrics (beyond the ones mentioned above) that are important to your application and send them to Sentry.
+
+For example, you might want to set a custom metric to track...
+
+<!-- need some use cases/examples here -->
```

Comment:
```suggestion
- Total memory usage during a transaction
- The amount of time being queried
- How many times a user performed an action during the transaction
```

Reply:
Re: @wmak's commit suggestion, we might want to say "during **a** transaction" for both examples 1 and 3 for consistency.

For example 2, is that number of queries or actual query time? Actual query time seems like it should be a span? 

It might be helpful if we include an example for each custom perf metric with their associated unit (if applicable).